mixer/test/client/route_directive/route_directive_test.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 // signifies a certain map has to be empty
-const mustBeEmpty = "MUST_BE_EMPTY"
+const mustBeEmpty = "Must-Be-Empty"
```

Comment:
This is a magic value that needs to be changed in all uses. Revert or change the examples, too?

Reply:
I don't see any other uses of MUST_BE_EMPTY in the source base, so I'm unclear what you're suggesting I change. Any pointers?

Reply:
Ok, I see, then it's OK I think. I mixed this one with another one used to indicate an empty field.